### PR TITLE
corrected link#

### PR DESCRIPTION
**Pull request by suzu9te**
I have corrected errors of links.

### CODE REVIEW

File: views/about.ejs [hunk 1]
```diff
@@ -95,7 +95,7 @@
 				<div class="span8">
 
 					<!-- start: About Us -->
-					<div id="about">
+					<div id="/about">
```
Comment (sueken5) [inline views/about.ejs]: これ違うよ

File: views/contact.ejs [hunk 2]
```diff
@@ -144,26 +144,26 @@
 								<div class="clearfix">
 									<label for="name"><span>Name:</span></label>
 									<div class="input">
-										<input tabindex="1" size="18" id="name" name="name" type="text" value="">
+										<input tab="1" size="18" id="name" name="name" type="text" value="">
```
Comment (sueken5) [inline views/contact.ejs]: ここも

File: views/about.ejs (hunk 1 above)
Comment (suzu9te) [reply views/about.ejs]: thanks

File: views/index.ejs [hunk 3]
```diff
@@ -28,7 +28,7 @@
 							</a>
 							<div class="nav-collapse collapse">
 								<ul class="nav">
-									<li class="active"><a href="index.html">Home</a></li>
+									<li class="active"><a href="/index">Home</a></li>
```
Comment (sueken5) [inline views/index.ejs]: wrong link

**Pull request merged**